setup.py

```diff
@@ -60,6 +59,7 @@ def package_files(directory):
         "PyQt5",
         "pyrsistent",
         "python-dateutil",
+        "pytz",
```

Comment:
The dep above this one, python-dateutil probably does both timezones and calculation/presentation of deltas. Did you check that library out before introducing pytz?

Reply:
I was under the impression (based on previous experience), that `pytz` was the go-to library for handling time zones. Looks like that was in the past and the `python-dateutil` has been considerably improved over the yeras - and in fact the `pytz` library now comes with caveats on how to use timezones (https://blog.ganssle.io/articles/2018/03/pytz-fastest-footgun.html). That was awkward (and in fact the way I had implemented it...). Changed now!